docs/outbound-transfers/Sandbox-Simulation.md

```diff
@@ -0,0 +1,15 @@
+---
+tags: [Outbound Transfers]
+---
+
+# Sandbox Simulation
+
+## Requesting an outbound transfer
+
+To request an outbound transfer, you will need to contact us. There is currently no API to request an outbound transfer.
```

Comment:
To speed this up, what would tenants will need to send us? I guess, just the account ID?